src/main/java/javax/measure/quantity/Miscellaneous.java

```diff
@@ -0,0 +1,61 @@
+package javax.measure.quantity;
+
+import javax.measure.converter.UnitConverter;
+import javax.measure.unit.BaseUnit;
+import javax.measure.unit.Unit;
+
+/**
+ * This interface represents a measure of miscellaneous quantities.
+ * In general units of this interface cannot be freely converted. This quantity is dimensionless.
+ *
+ * @author  <a href="mailto:ryan.smith@seeq.com">Ryan Smith</a>
+ * @version 4.3.4, March 2021
+ */
+public interface Miscellaneous extends Quantity {
+    public final static Unit<Miscellaneous> SG = new BaseUnit<>("SG");
+
+    public final static Unit<Miscellaneous> API = SG.transform(new SpecificGravityConverter(false));
+
+    public final static Unit<Miscellaneous> ppm = new BaseUnit<>("ppm");
+
+    public final static Unit<Miscellaneous> ppb = ppm.divide(1000);
+    /**
+     * This class represents the Specific Gravity Converter. See https://en.wikipedia.org/wiki/API_gravity.
+     */
```

Comment:
The first sentence of this doc is unhelpful (and a pet peeve of mine) as it just states the name of the class. While the Wikipedia link might be helpful, it's also pretty low effort to have it stand on its own. Try putting yourself in the shoes of someone who doesn't know what this is (like me) and explain to me why this super odd class is here.

Reply:
Good point. This came out of trying to enforce consistency within Jscience, but their coding standards are definitely not ours. Will clean that up.